mlir/lib/Dialect/AffineOps/AffineOps.cpp

```diff
@@ -134,8 +134,9 @@ bool mlir::isValidDim(Value value) {
     return false;
   }
   // This value has to be a block argument for a FuncOp or an affine.for.
-  auto *parentOp = value.cast<BlockArgument>().getOwner()->getParentOp();
-  return isa<FuncOp>(parentOp) || isa<AffineForOp>(parentOp);
+  // auto *parentOp = value.cast<BlockArgument>().getOwner()->getParentOp();
+  // return isa<FuncOp>(parentOp) || isa<AffineForOp>(parentOp);
+  return true;
```

Comment:
We should try to do this on a clean branch without my patch...

Reply:
Yes! I agree... This is what was there when I cloned master...

Reply:
Could you rebase this so that it's not on this branch?